lib/rubocop/cop/internal_affairs/expect_correction_if_autocorrect.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module InternalAffairs
+      # Checks for `expect_correction` or `expect_no_corrections` after `expect_offense`
+      # for cops that support autocorrect.
```

Comment:
Could you add to the documentation what determines if the cop supports autocorrect please?

Reply:
Sure, done